src/CMakeLists.txt

```diff
@@ -271,7 +271,7 @@ add_library(${src_objlib} OBJECT
     ${regions_files} 
     ${types_files} 
     ${utils_files}
-    ${examples_files} 
+    ${examples_files}
```

Comment:
I don't think you want example code in the library do you?

Reply:
these are the header files. I had a problem running the test that runs the (hotgym) example. But I guess I could add them to the example & tests, rather.